superbench/cli/_commands.py

```diff
@@ -23,6 +23,8 @@ def load_command_table(self, args):
             g.command('deploy', 'deploy_command_handler')
             g.command('exec', 'exec_command_handler')
             g.command('run', 'run_command_handler')
+        with CommandGroup(self, 'node', 'superbench.cli._node_handler#{}') as g:
```

Comment:
What's the meaning of this?

Reply:
create a command group https://github.com/microsoft/knack/blob/dev/docs/commands.md#commands